templates/Email/user.dataset-created.email.twig

```diff
@@ -7,7 +7,7 @@
     <br>
     The Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) has received your dataset submission {{ datasetSubmission.dataset.udi }}.
     The dataset landing page is <a href="{{ url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi}) }}">{{url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi})}}</a>.<br>
-    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi|default}} .<br><br>
+    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi.doi|default}} .<br><br>
```

Comment:
Maybe we should make another function called getIdentifier that only suplies the identifier, with doi cut off?

Reply:
getDoi() is already a function which does that

Reply:
I mean, one that is easier to read. Just a thought.